payment/controllers.go

```diff
@@ -445,6 +462,36 @@ func GetOrderCreds(service *Service) handlers.AppHandler {
 			)
 		}
 
+		order, err := service.Datastore.GetOrder(*orderID.UUID())
+
+		var credentials []TimeLimitedCreds
+		timeLimitedSecret := cryptography.NewTimeLimitedSecret([]byte(os.Getenv("BRAVE_MERCHANT_KEY")))
+		// FIXME - We should scope this to subscription paid date in case of paid order
+		issuedAt := order.CreatedAt
+		expiresAt := issuedAt.AddDate(0, 0, 35)
+		if order.IsPaid() {
+			for _, item := range order.Items {
+				if item.SKU == "brave-talk-free" || item.SKU == "brave-talk-paid" {
```

Comment:
this hack should be removed, we should be looking at what was specified in `credential_type` of the SKU token